kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -65,14 +125,26 @@ To run the generated native executable, generated in `target/`, execute
 
 ### Use the application
 
-To make use of this application it is as simple as putting a message on `travellers` topic with following content (cloud event format) 
+To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* To examine ProcessedTravellers topic and verify upcoming messages will be processed
```

Comment:
I would add to execute this in a different terminal

Reply:
fixed @radtriste